### ISSUE TITLE
Potential conflict on timestamps and their related "labels" in employee and project entries

### ISSUE THREAD

**Issue by paulflakstad** [npolar/api.npolar.no#41]
Employee and project entries suffer from a potential conflict between timestamps and their related "labels": 
* Employees: <code>NPI Employee</code> and <code>On Leave</code>
* Projects: <code>State</code>

These labels should ideally be "auto-updated" by the system, deduced from the timestamps. (As opposed to the current solution, which requires manual updating. <em>Alternatively, they could be dropped – they are, after all, redundant. However, they do make very useful filters.</em>)

Example situation: 
An employee can be set as "currently employed" by adding a "hired" timestamp, but the <code>NPI Employee</code> field could indicate the opposite, until manually updated. Same vice versa, and the same type of conflict exists for <code>On Leave</code>, as well as projects and their state (<code>planned</code>, <code>active</code>, etc.).

The current system effectively forces editors to continuously monitor all these timestamps/statuses manually, and manually update the entries. Furthermore, such updates must be done at very specific times (the exact times specified in the timestamps).

**Comment by cnrdh** [comment]
Yes, states like "planned" for completed projects (common) is awful...

These are "planned" just now...
N-ICE2015: Physical oceanography | project 
N-ICE2015: Logistics | project 
N-ICE2015: Atmosphere | project 
N-ICE2015: Outreach | project 
N-ICE2015: Ecosystem dynamics in a high-Arctic pack ice environment | project
